es: Improved translation on first two sections

aabreuglez opened the pull request:
Hi, I think the first two sections are more readable now.

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
 ![curl -s 'https://raw.githubusercontent.com/jlevy/the-art-of-command-line/master/README.md' | egrep -o '`\w+`' | tr -d '`' | cowsay -W50](cowsay.png)
 
-La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta mejora su flexibilidad y productividad como ingeniero en formas obvia y sutil. Esta es una selección de notas y consejos al usar el terminal que encontramos útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no es larga, pero ...
+La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta habilidad mejora tu flexibilidad y productividad como ingeniero de formas obvias y sutiles. Esta es una selección de notas y consejos para usar el terminal que encontramos útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no ...
```
habilidad:  Change the original idea, please, remove it.

alu0100601398 replied:
Removed  "habilidad" and changed from "esta" to "ésta" (is pronoum now).

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
 [apareció](http://www.quora.com/What-are-the-most-useful-Swiss-army-knife-one-liners-on-Unix)
 en [Quora](http://www.quora.com/What-are-some-time-saving-tips-that-every-Linux-user-should-know),
-pero debido al interés mostrado, parecía valer la pena usar Github, donde personas más talentosas que el autor original pudiendo fácilmente sugerir mejoras. Si ve un error o algo que podría ser mejor, por favor, crea un issue o PR! (Por supuesto primero revisa la sección meta de PRs/issues.)
+pero debido al interés mostrado, parecía valer la pena usar Github, donde personas más talentosas que el autor original pudieran sugergir mejoras fácilmente. Si encuentras un error o algo que podría mejorarse, por favor, ¡crea un issue o PR! (Por supuesto primero revisa la sección meta de PRs/issues.)
```
encuentras: It's different from the original verb used, i suggest use "ves"

alu0100601398 replied:
Changed to "ves".

AndresCidoncha replied:
In spanish, we use "encontrar un error" better than "ver un error".

ceoaliongroo replied:
@AndresCidoncha you are right, but still is a bad practice using the spanish, there is not a problem to use this verb in the sentence.

__From Real Academy Dictionary:__
http://lema.rae.es/drae/srv/search?val=ver

Also the most importan think keep the original idea and the spirit of the author.

We keep "ves" instead ëncontrar

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
-- Esta escrito para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos aplican o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
-- Se enfoca en Bash interactivo, aunque muchos de los consejos se aplican para otros shells y al Bash scripting por lo general.
-- Incluye tanto comandos "estándar" Unix, así como aquellos que requieren la instalación especial de un paquete -- siempre que sea suficientemente importante para ameritar su inclusión.
+- Esta guía es tanto para principiantes como para expertos. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente puede ahorrar tiempo comparado con otras alternativa ...
```
Let's keep "experimentados", please remove it.

alu0100601398 replied:
Sure, I probably changed the meaning with experts. I went back to "experimentados", but it's a bit weird. We use this word mostly related with sex. (No kidding).

AndresCidoncha replied:
I think that "experimentados" is almost not used. We use "Experto" or "Con experiencia".

ceoaliongroo replied:
Sorry is your perception.

The use "experimentado" is fine, also meaning expert

__From Real Academy Dictionary:__
http://lema.rae.es/drae/srv/search?id=SpNWvLZYBDXX2sftDQrf

AndresCidoncha replied:
I'm from Spain so I think that my perception is valid. "Experimentado" exist, but we don't use it.

ceoaliongroo replied:
Ok, i understood you, but is used in a lot of countries in Latina America. So in this case we keep "Experimentado" 

Thanks for your opinion

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
-- Se enfoca en Bash interactivo, aunque muchos de los consejos se aplican para otros shells y al Bash scripting por lo general.
-- Incluye tanto comandos "estándar" Unix, así como aquellos que requieren la instalación especial de un paquete -- siempre que sea suficientemente importante para ameritar su inclusión.
+- Esta guía es tanto para principiantes como para expertos. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente puede ahorrar tiempo comparado con otras alternativa ...
+- Está escrita para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos sirven o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
```
sirven: It's different from the original verb used. 

Maybe you could suggest a different verb, more related with emplear, usar, aplicar.

alu0100601398 replied:
I will let "aplican" by know. But it sounds really strange to me. I'll keep thinking about it.

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
-- Incluye tanto comandos "estándar" Unix, así como aquellos que requieren la instalación especial de un paquete -- siempre que sea suficientemente importante para ameritar su inclusión.
+- Esta guía es tanto para principiantes como para expertos. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente puede ahorrar tiempo comparado con otras alternativa ...
+- Está escrita para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos sirven o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
+- Se concentra en Bash interactivo, aunque muchos de los consejos sirven para otros shells y en general para el scripting en Bash.
```
concentra: please remove it.
sirven: It's different from the original verb used. Look the previous suggestion...  
"Bash scripting" is used without translation., this change not apply

alu0100601398 replied:
Sorry about that , I meant "centra".
I will let "aplican" here too.
Back to Bash scripting.

ceoaliongroo replied:
Don't changes the original idea of the sentence, please keep:
`"... al Bash scripting por lo general."`

ceoaliongroo replied:
Please remove:
`como algunos que requieren la instalación de un paquete especial  `
And add:
`..así como los que requieren instalaciones especiales de paquetes`

ceoaliongroo replied:
Nice done with "centrar" and "merecer"

alu0100601398 replied:
Hi! Thank you so much
Back to the first version on the first change.
But in the second one:
"special package installs"
This refers to "special packages", isn't it?
I changed it to "..así como los que requieren instalaciones de paquetes especiales" .
And as they are in plural, I changed the next sentence to plural too.
"siempre que sean lo suficientemente importantes"

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
+- Esta guía es tanto para principiantes como para expertos. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente puede ahorrar tiempo comparado con otras alternativa ...
+- Está escrita para Linux, con excepción de la sección "[Solo para OS X](#os-x-only)". Muchos de los otros puntos sirven o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
+- Se concentra en Bash interactivo, aunque muchos de los consejos sirven para otros shells y en general para el scripting en Bash.
+- Incluye tanto comandos "estándar" Unix como algunos que requieren la instalación de un paquete especial -- siempre que sea lo suficientemente importante para justificar su inclusión.
```
Please change to:  

Incluye tanto comandos "estándar" de Unix así como algunos que requieren la instalación de un paquete especial -- siempre que sea suficientemente importante para ameritar su inclusión.

alu0100601398 replied:
I've never readed before "ameritar" I changed it to "merecer".

ceoaliongroo replied:
:+1:

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
 
-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
-- Usa [Explainshell](http://explainshell.com/) para obtener detalles de ayuda sobre que comandos, opciones, pipes, etc.
+- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares buscando en Google una vez conoces la idea o el comando. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
```
I don't see this is necessary

alu0100601398 replied:
I changed it back. I moved the comma.

ceoaliongroo replied:
It's not necessary, please remove the changes.

alu0100601398 replied:
Hi, I removed the changes, but it's really hard to read that sentence. I'll keep thinking about that.

ceoaliongroo replied:
I did a double check and you are right

Please reprhase to:
 ...otros lugares buscando en Google una vez conoces la idea o el comando.

aabreuglez replied:
Done, much better now.

ceoaliongroo commented on README-es.md:
```diff
@@ -21,29 +21,29 @@
-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
-- Usa [Explainshell](http://explainshell.com/) para obtener detalles de ayuda sobre que comandos, opciones, pipes, etc.
+- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares buscando en Google una vez conoces la idea o el comando. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
+- Usa [Explainshell](http://explainshell.com/) para obtener una ayuda detallada sobre que hacen los comandos, las opcioneslas pipes, etc.
```
I don't see this is necessary

alu0100601398 replied:
Changed the paragraph back. And now modified to: "que hacen los comandos, las opciones, las pipes, etc."

ceoaliongroo replied:
It's not necessary, please remove the changes.

alu0100601398 replied:
Hi,but in the original version appears the word "do" -> Use Explainshell to get a helpful breakdown of what commands, options, pipes etc. do.
Can you check it?

ceoaliongroo replied:
Good, 

Please change:
"las opcioneslas pipes," => "opciones, pipes"

aabreuglez replied:
I  change the phrase to:
"Usa Explainshell para obtener detalles de ayuda sobre que hacen los comandos, las opciones, las pipes, etc."

Tried:
"sobre que hacen comandos, opciones, pipes, etc". But sounds a bit strange. What do you think?

ceoaliongroo commented:
@alu0100601398 Thanks you very much for you collaboration. 

Please review the suggestion and do changes.

alu0100601398 commented:
@ceoaliongroo Thanks you for reviewed everything!
I did changes, and I commented all of them.

ceoaliongroo commented:
@alu0100601398 I add more comment, please review it.

Thanks

alu0100601398 commented:
Hi @ceoaliongroo I reviewed the changes. I let two comments for you.
Thank you for your time.

ceoaliongroo commented:
@aabreuglez good job, 

please after do the change merge with the master, to do the final check

thanks

ceoaliongroo commented on README-es.md:
```diff
@@ -36,14 +36,14 @@ pero debido al interés mostrado, parecía valer la pena usar Github, donde pers
 
-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
-- Usa [Explainshell](http://explainshell.com/) para obtener detalles de ayuda sobre que comandos, opciones, pipes, etc.
+-- Para mantener esto en una página, el contenido está incluido implícitamente por referencia. Eres lo suficientemente inteligente para consultar más detalles en otros lugares, cuando conoces la idea o comando con Google. Usa `apt-get`/`yum`/`dnf`/`pacman`/`pip`/`brew` (según proceda) para instalar los nuevos programas.
```
Remove extra dash.

jlevy commented:
Let me know if/how this gets resolved. Always good to keep the improvements coming in! Thanks for the efforts!

ceoaliongroo commented:
@aabreuglez, do you need help to do the changes and merge with master?

aabreuglez commented:
Sorry, @ceoaliongroo not much free time these days (exams season).
Merged the changes, updated. Fixed.
Check a note I let you
Also added you as a collaborator.

Regards

ceoaliongroo commented:
@aabreuglez Great!, thanks for take the time to do the changes. Good luck in the exams!

@jlevy, it's ready to merge, LGTM

jlevy commented:
:+1: Thanks for the teamwork, @aabreuglez @ceoaliongroo !
